source/acquaman/IDEAS/IDEASXASScanActionController.cpp

```diff
@@ -232,7 +238,9 @@ AMAction3* IDEASXASScanActionController::createCleanupActions()
 {
 	AMListAction3 *cleanupActions = new AMListAction3(new AMListActionInfo3("IDEAS XAS Cleanup Actions", "IDEAS XAS Cleanup Actions"));
 
-	cleanupActions->addSubAction(IDEASBeamline::ideas()->scaler()->createDwellTimeAction3(0.1));
+	cleanupActions->addSubAction(new AMWaitAction(new AMWaitActionInfo(IDEASBeamline::ideas()->scaler()->dwellTime())));
```

Comment:
I know the BioXAS scaler uses strategic waits when manipulating that PV, but for us it toggles between 'Armed/Disarmed' states instead of 'SingleShot/Continuous.' But the wait is still there. Maybe we should take another look at this scaler behavior, as part of another issue?